src/main/resources/application.yaml

```
@@ -0,0 +1 @@
+spring.application.name=discodeit
```

properties에서 yaml로 바꿨다면 형식도 맞게 변경하면 좋을 것 같아요.  
```suggestion
spring:
  application:
    name: discodeit
```